src/System.Private.CoreLib/src/System/Runtime/InteropServices/Marshal.cs

```diff
@@ -12,50 +12,35 @@
 **
 =============================================================================*/
 
+using System.Collections.Generic;
+using System.Reflection;
+using System.Reflection.Emit;
+using System.Security;
+using System.Text;
+using System.Runtime.CompilerServices;
+using System.Runtime.ConstrainedExecution;
+using Win32Native = Microsoft.Win32.Win32Native;
+using System.Diagnostics;
+using System.Runtime.InteropServices.ComTypes;
+using System.StubHelpers;
+
 namespace System.Runtime.InteropServices
 {
-    using System;
-    using System.Runtime;
-    using System.Collections.Generic;
-    using System.Reflection;
-    using System.Reflection.Emit;
-    using System.Security;
-    using System.Text;
-    using System.Threading;
-    using System.Runtime.CompilerServices;
-    using System.Globalization;
-    using System.Runtime.ConstrainedExecution;
-    using System.Runtime.Versioning;
-    using Win32Native = Microsoft.Win32.Win32Native;
-    using Microsoft.Win32.SafeHandles;
-    using System.Diagnostics;
-    using System.Runtime.InteropServices.ComTypes;
-    using System.StubHelpers;
-
     public enum CustomQueryInterfaceMode
     {
         Ignore = 0,
         Allow = 1
     }
 
-    //========================================================================
-    // All public methods, including PInvoke, are protected with linkchecks.  
-    // Remove the default demands for all PInvoke methods with this global 
-    // declaration on the class.
-    //========================================================================
-
     public static partial class Marshal
     {
-        //====================================================================
-        // Defines used inside the Marshal class.
-        //====================================================================
         private const int LMEM_FIXED = 0;
         private const int LMEM_MOVEABLE = 2;
 #if !FEATURE_PAL
-        private const long HIWORDMASK = unchecked((long)0xffffffffffff0000L);
+        private const long HiWordMask = unchecked((long)0xffffffffffff0000L);
 #endif //!FEATURE_PAL
 #if FEATURE_COMINTEROP
-        private static Guid IID_IUnknown = new Guid("00000000-0000-0000-C000-000000000046");
+        private static Guid s_iidIUnknown = new Guid("00000000-0000-0000-C000-000000000046");
```

Comment:
Please revert this format change. It is the same identifier in native and makes searching for usage across the code base much easier.